string msgType
string agvId
int64 timestamp
# recvtask_value value
int32 taskId
int32 subTaskIndex



